src/test/run-pass/issue-3121.rs

```diff
@@ -8,7 +8,7 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-// pretty-expanded FIXME #23616
+// no-pretty-expanded FIXME #23616 assert_receiver_is_clone
```

Comment:
Fair enough. Does that mean when I write a new run-pass test I should try
to add "// pretty-expanded"? Do we have a policy on that?

On Mon, Apr 25, 2016 at 2:24 PM, Alex Crichton <notifications@github.com>
wrote:

> In src/test/run-pass/issue-3121.rs
> <https://github.com/rust-lang/rust/pull/31414#discussion_r60962453>:
>
> > @@ -8,7 +8,7 @@
> >  // option. This file may not be copied, modified, or distributed
> >  // except according to those terms.
> >
> > -// pretty-expanded FIXME #23616
> > +// no-pretty-expanded FIXME #23616 assert_receiver_is_clone
>
> Nah yeah it's fine to just remove everywhere, right now that phase of
> testing is opt-in
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/rust-lang/rust/pull/31414/files/64d22d453d5b71b33b149dac3c28b2d626446d9f#r60962453>
>